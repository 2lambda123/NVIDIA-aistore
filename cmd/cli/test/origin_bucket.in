ais bucket create $BUCKET

ais bucket props ais://$BUCKET backend_bck=ais://something // FAIL "Backend bucket "ais://something" must be a cloud bucket"
ais bucket props ais://$BUCKET backend_bck=gcp:// // FAIL "Invalid format "backend_bck.name" cannot be empty when "backend_bck.provider" is set"

ais show bucket-props ais://$BUCKET backend_bck -v
