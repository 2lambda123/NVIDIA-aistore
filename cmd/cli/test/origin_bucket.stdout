"$BUCKET" bucket created
PROPERTY         VALUE
provider         gcp
PROPERTY            VALUE
backend_bck.name
backend_bck.provider
Bucket props successfully updated
"backend_bck.name" set to:"ais-nv" (was:"")
"backend_bck.provider" set to:"gcp" (was:"")
 PROPERTY                 VALUE
backend_bck.name         ais-nv
backend_bck.provider     gcp
Bucket props successfully updated
"backend_bck.name" set to:"" (was:"ais-nv")
"backend_bck.provider" set to:"" (was:"gcp")
 PROPERTY               VALUE
backend_bck.name
backend_bck.provider
